src/stream/src/task/stream_manager.rs

```diff
@@ -623,38 +581,54 @@ impl LocalStreamManagerCore {
             let actor = Actor::new(
                 dispatcher,
                 subtasks,
+                actor,
                 self.context.clone(),
                 self.streaming_metrics.clone(),
                 actor_context.clone(),
                 expr_context,
             );
 
+            ret.push(actor);
+        }
+        Ok(ret)
+    }
+}
+
+impl LocalStreamManagerCore {
+    fn spawn_actors(
+        &mut self,
+        actors: Vec<Actor<DispatchExecutor>>,
+        streaming_metrics: &Arc<StreamingMetrics>,
+    ) {
+        for actor in actors {
             let monitor = tokio_metrics::TaskMonitor::new();
+            let actor_id = actor.stream_actor.actor_id;
 
             let handle = {
-                let context = self.context.clone();
-                let actor = async move {
-                    if let Err(err) = actor.run().await {
+                let trace_span = format!(
+                    "Actor {actor_id}: `{}`",
+                    actor.stream_actor.mview_definition
```

Comment:
Maybe just include the `mview_name` here? Some `mview_definitions` can be really large.

Reply:
Oh I see, previously we did this as well. Can ignore my comment. Out of scope.

Reply:
Previously we used the `mview_definition`. Shall we change to use the `mview_name`? cc @BugenZhao

Reply:
I guess `mview_definition` is actually `mview_name` in production builds. 😄

https://github.com/risingwavelabs/risingwave/blob/023a7fdba716d229614d266d884b3262c4370f1f/src/meta/src/stream/stream_graph/actor.rs#L282-L286